core/core_builders.py

```diff
@@ -30,6 +30,7 @@ def version_info_builder(target, source, env):
 #define GODOT_VERSION_WEBSITE "{website}"
 #define GODOT_VERSION_DOCS_BRANCH "{docs_branch}"
 #define GODOT_VERSION_DOCS_URL "https://docs.godotengine.org/en/" GODOT_VERSION_DOCS_BRANCH
+#define GODOT_VERSION_CONTRIBUTING_DOCS_URL "https://contributing.godotengine.org/en/latest"
```

Comment:
`GODOT_VERSION_CONTRIBUTING_DOCS_URL` isn't the best name for this, as this is technically *removed* from the docs now. Maybe `GODOT_VERSION_CONTRIBUTE_URL` would be preferable, as it'd match the naming scheme `CONTRIBUTE_URL` in `editor_help.cpp`. Despite that, it should still pull the versioning from `GODOT_VERSION_DOCS_BRANCH`
```suggestion
#define GODOT_VERSION_CONTRIBUTE_URL "https://contributing.godotengine.org/en/" GODOT_VERSION_DOCS_BRANCH
```